sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -5454,6 +5461,8 @@ class SQLConf extends Serializable with Logging with SqlApiConf {
   def disabledV2StreamingMicroBatchReaders: String =
     getConf(DISABLED_V2_STREAMING_MICROBATCH_READERS)
 
+  def disableMapKeyNormalization: Boolean = getConf(DISABLE_MAP_KEY_NORMALIZATION)
```

Comment:
If it's only called by one place, we can just inline it.

Reply:
added `@inline` - or do we just inline it manually where its called?

Reply:
I mean manually inline it. Do not create this method.